src/codeflare_sdk.egg-info/PKG-INFO

```diff
@@ -1,4 +1,4 @@
 Metadata-Version: 2.1
-Name: codeflare-sdk
+Name: codeflare_sdk
```

Comment:
```suggestion
Name: codeflare-sdk
```
Perhaps if you could add this file to the `.gitignore`?

Reply:
Perhaps the whole `codeflare_sdk.egg-info` directory should be in the .gitignore. Then we could close: https://issues.redhat.com/browse/RHOAIENG-6635

Reply:
Yes it is possible to remove tracked file and add ignore it.